### ISSUE TITLE
Renderer Support for item.initial/nestedItems

### ISSUE THREAD

**Issue by patrick-werner** [aehrc/smart-forms#590]
I tried to use: 
- item.initital.valueString
- item.item (nested items)
With the Smart Forms Renderer, also with the playground: https://www.smartforms.io/playground both functionalities aren't working.

test questionnaire: 
https://fhir.hl7.de/fhir/Questionnaire/1700

**Comment by fongsean** [comment]
Thanks for reporting this!

Item.initial isn't supported yet as per https://github.com/aehrc/smart-forms/blob/main/Conformance.md.
Seems like we only support item.item if the parent item is of type "group". I might head over to Zulip and get some advice on how it should be rendered.

**Comment by patrick-werner** [comment]
Thanks for the reply. I think initial would be a nice feature to support and also very easy to implement.

**Comment by fongsean** [comment]
I have implemented a rough implementation for item.initial and item.answerOption.initialSelected.

item.initial should work with the https://smartforms.csiro.au/api/fhir/Questionnaire/Test1700 questionnaire. Notice I have added the initial attribute in Item "1.2".

Should be working with repeat items, still a little flaky with repeat groups at the moment but I will continue to iterate from there. Let me know what you think!

Will keep the implementation of nesting items on my list.
